### PR TITLE
Add date function aliases

### PR DESCRIPTION
**Pull request by papparapa**
Added some  function aliases docs for `TIMESTAMP`,  `TIMESTAMPTZ`, `INTERVAL`, and `TIME`.

### CODE REVIEW

File: docs/sql/functions/timestamp.md [hunk 1]
```diff
@@ -26,10 +26,15 @@ The table below shows the available scalar functions for `TIMESTAMP` values.
...
+| `datediff(`*`part`*`, `*`startdate`*`, `*`enddate`*`)` | Alias of date_diff. The number of [partition](../../sql/functions/datepart) boundaries between the timestamps | `datediff('hour', TIMESTAMP '1992-09-30 23:59:59', TIMESTAMP '1992-10-01 01:58:00')` | `2` |
 | `date_part(`*`part`*`, `*`timestamp`*`)` | Get [subfield](../../sql/functions/datepart) (equivalent to *extract*) | `date_part('minute', TIMESTAMP '1992-09-20 20:38:40')` | `38` |
+| `datepart(`*`part`*`, `*`timestamp`*`)` | Alias of date_part. Get [subfield](../../sql/functions/datepart) (equivalent to *extract*) | `datepart('minute', TIMESTAMP '1992-09-20 20:38:40')` | `38` |
 | `date_part([`*`part`*`, ...], `*`timestamp`*`)` | Get the listed [subfields](../../sql/functions/datepart) as a `struct`. The list must beconstant. | `date_part(['year', 'month', 'day'], TIMESTAMP '1992-09-20 20:38:40')` | `{year: 1992, month: 9, day: 20}` |
+| `datepart([`*`part`*`, ...], `*`timestamp`*`)` | Alias of date_part. Get the listed [subfields](../../sql/functions/datepart) as a `struct`. The list must beconstant. | `datepart(['year', 'month', 'day'], TIMESTAMP '1992-09-20 20:38:40')` | `{year: 1992, month: 9, day: 20}` |
```
Comment (hawkfish) [inline docs/sql/functions/timestamp.md:33]: be constant

Comment (hawkfish) [reply docs/sql/functions/timestamp.md:33]: Please fix the one you copied too (my bad I think!)

Comment (papparapa) [reply docs/sql/functions/timestamp.md]: Oops I didn't notice that

**Comment by Alex-Monahan** [comment]
Thank you to @papparapa for the PR and @hawkfish for the review! Always nice to be as compatible as possible when it is as simple as an underscore difference, and even better to have it documented... Cheers!

**Pull request merged**